tests/unit/AvatarWrapper.spec.js

```diff
@@ -0,0 +1,57 @@
+import { shallowMount } from '@vue/test-utils'
```

Comment:
didn't we agree on having hte test files right next to the components they are testing?

Reply:
I haven't been able to config it so that Jest look into the whole src folder structure

Reply:
https://github.com/nextcloud/text/blob/master/package.json#L22

Reply:
We can't do that because we're using @vue/unit-jest, not jest itself. I solved it in the last commit https://github.com/nextcloud/spreed/pull/3078/commits/fcc4dbb49f2c6b7c487db1b27afc4ea251115fe4